rioxarray/rioxarray.py

```diff
@@ -894,7 +894,7 @@ def bounds(self, recalc=False):
 
         return left, bottom, right, top
 
-    def isel_window(self, window):
+    def isel_window(self, window, pad=False):
         """
         Use a rasterio.window.Window to select a subset of the data.
 
```

Comment:
Mind adding `.. versionadded:: 0.6.0 pad` here?

Reply:
done!

Reply:
Thanks - I am not seeing the change.